client/ayon_core/pipeline/workfile/workfile_template_builder.py

```diff
@@ -313,7 +313,8 @@ def _get_linked_folder_entities(self):
         if not folder_entity:
             return []
         links = get_folder_links(
-            project_name, folder_entity["id"], link_direction="in"
+            project_name,
+            folder_entity["id"], link_types=["template"], link_direction="in"
```

Comment:
Shouldn't be the link type as combobox part of template builder UI?

Reply:
I was thinking about it and frankly, I am not sure. It would be more flexible then I guess, but also mixing link types could really mess things?

Reply:
I think this makes it unusable. If you already have links in AYON but with different link type, you have to recreate them again.